observability/network_observability/netobserv_cli/netobserv-cli-reference.adoc

```diff
@@ -0,0 +1,10 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="netobserv-cli-reference"]
+= netobserv CLI reference
+include::_attributes/common-attributes.adoc[]
+:context: cli-reference
+
+toc::[]
+Many of the Network Observability Operator's features work in the CLI. You can pass command line arguments to enable features or pass configuration options to the eBPF agent.
```

Comment:
The features Drop / DNS / RTT are all available. You can get the same data as with the operator. The main difference is the way of storing content which avoid you to make graphs on top of the collected data.

Not available features:
- storage & exporters (on purepose)
- console plugin
- dashboards

```suggestion
Most of the Network Observability Operator's features work in the CLI. You can pass command line arguments to enable features or filtering options.
```

FYI the eBPF agent also embed flowlogs-pipeline (processor) for enrichment in a single pod per node using the CLI. That's something we are considering for Operator Direct deployment too.

Reply:
@msherif1234 For 4.12 CLI, will packet drop work?

Reply:
no 4.12 has older kernel `RHEL8.4` that won't support pkt drop with cli or operator nothing todo with the CLI

Reply:
Hmmmmm...ok thanks for confirming. In this case, maybe I can merge this PR to 4.13+ and do a manual cherry-pick to 4.12, except remove the packet drop. Not necessarily create an entirely separate PR, since there's only a couple mentions of packet drops.

Reply:
SO what I'm going to do here is merge this to `no-1.6`. `no-1.6` will go to all versions of OCP (4.12+). So then once everything is up, I'm going to open a PR to effectively remove the packet drop info from 4.12 only. I think that's the easiest way to do it.

Reply:
For posterity, the PR where I remove packet drop in 4.12: https://github.com/openshift/openshift-docs/pull/77518/files#diff-c18247ebe497a5c92e79ac355fd76da99dc3ee86cd685775eb99c32e11eea0ceR37-R47